poi.config.js

```diff
@@ -85,7 +85,8 @@ module.exports = {
   },
   envs: {
     API_PATH: '/api/v1/',
-    AUTH_JWT_SCHEME
+    AUTH_JWT_SCHEME,
+    PERSIST_STORAGE_KEY: 'TAILOR_USER'
```

Comment:
Consider updating key name / value since:
- Other modules can be added later on (with new approach) so I am not sure if TAILOR_USER name makes sense (rather go with something more generic like) APP_STATE / APP_STORE ?
- For me PERSIST_STORAGE_KEY isn't descriptive enough (what is it actually being persisted) so I would rename it to something like VUEX_STORAGE_KEY

Reply:
Do you mind if I add a short comment above `VUEX_STORAGE_KEY: APP_STORE` describing the purpose of this setting. 

It would be something like: 
> The key to store the Vuex state in the storage

Reply:
Sure 👍

Reply:
Quoting myself https://github.com/ExtensionEngine/tailor/pull/367#discussion_r353490268:
>My vote goes for TAILOR_STATE or even better TAILOR_APP_STATE because:
>
>1. you really want to prefix it with app name cause otherwise it will potentially clash with other locally developed apps running on same port
>2. seing state suffix indicates that it came from vuex